lib/snippets/push-to-heroku

```diff
@@ -62,11 +62,12 @@ if [ "$MIGRATIONS" != "" ]; then
     exit 1
   fi
 
-  green "Post-migration dyno restart."
-  heroku restart --app $HEROKU_APP
+  if [ -z $NO_POST_MIGRATION_RESTART ]; then
```

Comment:
Google's Shell styleguide is a good reference to write robust scripts. Notably in your case:
  - use `if [[ cond ]]` instead of `if [ cond ]`: https://google.github.io/styleguide/shell.xml?showone=Test,_%5B_and_%5B%5B#Test,_%5B_and_%5B%5B
  - and always interpolating variables `"${BLAH}"` instead of `$BLAH`: https://google.github.io/styleguide/shell.xml?showone=Testing_Strings#Testing_Strings